benchmarks/jmh/src/jmh/kotlin/com/linecorp/armeria/internal/common/kotlin/AnnotatedServiceFlowBenchmark.kt

```diff
@@ -37,32 +37,32 @@ import java.util.stream.IntStream
 @Warmup(iterations = 1)
 @Suppress("unused")
 open class AnnotatedServiceFlowBenchmark {
-
     lateinit var server: Server
     lateinit var client: WebClient
 
     @Setup
     open fun setup() {
-        server = Server.builder()
-            .annotatedService(
-                "/benchmark",
-                object {
-                    @Get("/flow")
-                    @ProducesJsonSequences
-                    fun flowBm(): Flow<String> = flow {
-                        (0 until 1000).forEach {
-                            emit("$it")
-                        }
-                    }
+        server =
+            Server.builder()
+                .annotatedService(
+                    "/benchmark",
+                    object {
+                        @Get("/flow")
+                        @ProducesJsonSequences
+                        fun flowBm(): Flow<String> =
+                            flow {
+                                (0 until 1000).forEach {
+                                    emit("$it")
+                                }
+                            }
 
-                    @Get("/publisher")
-                    @ProducesJsonSequences
-                    fun publisherBm(): Publisher<String> =
-                        Flux.fromStream(IntStream.range(0, 1000).mapToObj { it.toString() })
-                }
-            )
-            .build()
-            .also { it.start().join() }
+                        @Get("/publisher")
+                        @ProducesJsonSequences
+                        fun publisherBm(): Publisher<String> = Flux.fromStream(IntStream.range(0, 1000).mapToObj { it.toString() })
```

Comment:
Don't they have line length limit? 🤔

Reply:
https://pinterest.github.io/ktlint/latest/rules/standard/#max-line-length
The maximum length of `ktlint_official` is 140.
It would be good to limit the value to `112` to abide by our convention.

Reply:
Create #5421 for that.